static/js/modals.js

```diff
@@ -309,3 +309,26 @@ function getCustomFields(event) {
   const textarea = document.getElementById("Comments_from_lead__c");
   textarea.value = message;
 }
+
+// Fix Tab and modal js conflicts
+(function () {
```

Comment:
Haven't looked into it, but would it be possible to add a check to the initial ['aria-controls' listener for forms](https://github.com/canonical/canonical.com/blob/main/static/js/modals.js#L138) to see if it has an associated form (with `document.getElementById(targetControls)`) before calling `toggleModal`, so it doesn't affect other elements with aria-controls?